video_embed_field.feeds.inc

```diff
@@ -53,24 +53,22 @@ function video_embed_field_set_target($source, $entity, $target, $value) {
   $field = isset($entity->$field_name) ? $entity->$field_name : array(LANGUAGE_NONE => array());
 
   // Allow for multiple mappings to the same target.
-  if (!empty($field[LANGUAGE_NONE])) {
-    $count = call_user_func_array('array_merge_recursive', $field[LANGUAGE_NONE]);
-    $delta = count($count[$sub_field]);
-  }
-  else {
-    $delta = 0;
-  }
+  $field[LANGUAGE_NONE] = !empty($entity->{$field_name}[LANGUAGE_NONE]) ? $field[LANGUAGE_NONE] : [[]];
```

Comment:
```suggestion
  $field[LANGUAGE_NONE] = $field[LANGUAGE_NONE] ?? [[]];
```

Reply:
@elisseck Does this do the same thing? With the release this week I think we can start using `??` as PHP 5.x support is officially dropped.

Reply:
@laryn yup, I forgot we could maybe use php 7 stuff now. That will work and is an easier read. Thanks!